.babelrc

```diff
@@ -1,13 +1,19 @@
 {
   "sourceMaps": true,
   "presets": [
-    "es2015-minimal",
+    "es2015",
```

Comment:
@stephencookdev if you remember asking me about the errors of  `resolve failed:` this is what was causing it.